Application/EdFi.Ods.AdminApi/Features/ClaimSets/ClaimSetModel.cs

```diff
@@ -33,7 +31,7 @@ public class ResourceClaimModel
     public bool Read { get; set; }
     public bool Update { get; set; }
     public bool Delete { get; set; }
-    [SwaggerExclude(EdFiOdsSecurityModelCompatibility.ThreeThroughFive)]
+    //[SwaggerExclude(EdFiOdsSecurityModelCompatibility.ThreeThroughFive)]
```

Comment:
@DavidJGapCR 
Please remove the commented lines. Please make the changes on Phase2 - part 2